src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/ConsentOrderApprovedController.java

```diff
@@ -88,7 +91,7 @@
                 .orderLetter(approvedConsentOrderLetter)
                 .consentOrder(consentOrderAnnexStamped);
 
-            if (approvedConsentOrderNotificationLetterFeature) {
+            if (featureToggleService.isFeatureEnabled(APPROVED_CONSENT_ORDER_NOTIFICATION_LETTER)) {
```

Comment:
can we please add more methods:

```featureToggleService.isApprovedConsentOrderNotificantionEnabled()```

In my opinion it looks better

Reply:
Agreed - changes made 👍